lib/util/mutil.ml

```diff
@@ -506,13 +507,23 @@ let input_lexicon lang ht open_fname =
              && String.unsafe_get line 2 = ':'
              && String.unsafe_get line 3 = ' '
         then
+        begin
           let k2 = String.sub line 4 (String.length line - 4) in
-          Opt.iter (Hashtbl.replace ht k) (Hashtbl.find_opt ht k2) ;
+          Hashtbl.replace tmp k k2;
           key ()
+        end
         else trad k
-      | None -> key ()
+      | None -> key () ;
   in
-  key ()
+  key () ;
+  if Hashtbl.length tmp > 0 then
```

Comment:
You don't need this condition. `Hashtbl.iter` on an empty `Hashtbl.t` will just do nothing.